From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Mar 2023 23:34:50 +0000
Subject: driver: Set a non-trivial window title

When running in "existing user" mode (which isn't done upstream, but
Debian and Ubuntu re-enable it because they don't have GNOME Tour),
the window title is visible in Alt+[key above Tab] and the Overview.
It defaults to "gnome-initial-setup", but looks better if we reuse the
translated name from the .desktop file.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/merge_requests/191
---
 gnome-initial-setup/gis-driver.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gnome-initial-setup/gis-driver.c b/gnome-initial-setup/gis-driver.c
index 1ddf3c6..8459efa 100644
--- a/gnome-initial-setup/gis-driver.c
+++ b/gnome-initial-setup/gis-driver.c
@@ -797,6 +797,7 @@ gis_driver_startup (GApplication *app)
                                     "application", app,
                                     "icon-name", "preferences-system",
                                     "deletable", FALSE,
+                                    "title", _("Initial Setup"),
                                     NULL);
 
   g_signal_connect (driver->main_window,
